deploy/template.yaml

```diff
@@ -25,6 +25,63 @@ Conditions:
       -  0
 
 Resources:
+
+  IPVCriUKPassportPrivateAPI:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: !Sub IPV Passport Back Private API Gateway ${Environment}
+      EndpointConfiguration:
+        Type: PRIVATE
+      Auth:
+        ResourcePolicy:
+          CustomStatements:
+            - Action: 'execute-api:Invoke'
+              Effect: Allow
+              Principal: '*'
+              Resource:
+                - 'execute-api:/*'
+            - Action: 'execute-api:Invoke'
+              Effect: Deny
+              Principal: '*'
+              Resource:
+                - 'execute-api:/*'
+              Condition:
+                StringNotEquals:
+                  'aws:SourceVpce': !If
+                    - IsDevelopmentEnvironment
```

Comment:
`IsDevelopmentEnvironment` condition is not defined in the `Conditions` sections. I highly recommend installing cfn-lint in VScode...